docs/internal-documentation/maintenance/Garbage-Collection-Testing.md

```diff
@@ -0,0 +1,122 @@
+# Testing instructions for the Garbage Collection feature of the Zonemaster Backend module
+
+## Introduction
+The purpose of this instruction is to serve as a notice for manual testing of the new garbage collection feature at release time.
+
+## Testing the unfinished tests garbage collection feature
+
+1. Ensure that the database has the required aditionnal columns for this feature:
+```
```

Comment:
The indentation gets messed up here (and in many places below) when the document is rendered to HTML. You probably want to indent this with three spaces and check the result using the preview feature.

Reply:
Fixed in the commit 65f5257